messages.pot

```diff
@@ -1,14 +1,14 @@
 # Translations template for xl_auth.
-# Copyright (C) 2017 National Library of Sweden
+# Copyright (C) 2018 National Library of Sweden
```

Comment:
Autoupdate :(

Reply:
Yeah.. I'm thinking "let's keep it as this Babel tool wants it", else we'll have to go revert it manually on each `flask translate`. Seems like a waste of time..